src/iuse_actor.cpp

```diff
@@ -361,8 +361,24 @@ void consume_drug_iuse::load( JsonObject &obj )
 
 void consume_drug_iuse::info( const item&, std::vector<iteminfo>& dump ) const
 {
+    std::string vits;
+    for( const auto &v : vitamins ) {
+        // only display vitamins that we actually require
+        if( g->u.vitamin_rate( v.first ) > 0 ) {
+            if( !vits.empty() ) {
+                vits += ", ";
+            }
+            int lo = int( v.second.first  / ( DAYS( 1 ) / float( g->u.vitamin_rate( v.first ) ) ) * 100 );
+            int hi = int( v.second.second / ( DAYS( 1 ) / float( g->u.vitamin_rate( v.first ) ) ) * 100 );
```

Comment:
Also, I'd assign `g->u.vitamin_rate( v.first ) )` to a temp. variable since the invocation isn't free (it iterates over mutations) and there's no guarantee that compilers optimize that. Besides, it would look nicer.